packages/material-ui/src/styled/styled.js

```diff
@@ -0,0 +1,31 @@
+import seStyled from '@material-ui/styled-engine';
```

Comment:
What would be the downside of re-exporting `/packages/material-ui/src/styles/muiStyled.js` raw (and merge the two)? I can only see the upsides: gaining access to our theming engine.

Reply:
`muiStyled` solves different problem and should be used as an internal utility - it knows how to handle name from the theme and apply theme overrides and variants on top of the specified styles. I wouldn't mixed the two to be honest ( I wouldn't expose `muiStyled` for externala usage - if it is exported it is not intended)

Reply:
What downside do you envision? We have developers using withStyles with all its capability to build custom components and design system, e.g. leveraging the name.

Reply:
I see, using `muiStyled` on developers side can help them creating their custom components. My thought was we may decide to change the API, if it is private it would  be easier, but as we are in alpha that shouldn't be a problem. Making the third arguument optional in the `muiStyled` could help towards having one utility. We just have to make sure when using it to create our own compoenents we must specify it.. Is this what you are suggesting?

Reply:
Added the additional functionality in `muiStyled`, there is not really need for adding additional utility 👍